RomanNumeralsCI/src/main/java/com/ericsson/peterbuki/RomanConverter.java

```diff
@@ -4,43 +4,83 @@
  * Created by epetbuk on 2016.05.18..
  */
 public class RomanConverter {
-
+    public static final int MAX_VALUE = 3999 ;
 
     protected RomanConverter()
     {
         System.out.println("RomanConverter constructor.") ;
-        for (RomanNumber n : RomanNumber.values())
+        for (RomanNumberEnum n : RomanNumberEnum.values())
         {
             System.out.println(n.name + " = " + n.value) ;
         }
     }
 
     public static int convertToArabic(String in)
     {
+        boolean wasLastDouble ;
         int result = 0 ;
-        String toDecode ;
-        for (int position = 0; position < in.length(); position++)
+        for (int position = 0; position < in.length(); position+=wasLastDouble?2:1)
```

Comment:
this `for()` loop is hard too read. I kind of know what you want to do, but the `?:` makes it hard to read.